packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTableSet.tsx

```diff
@@ -0,0 +1,292 @@
+import * as React from 'react';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+import { Text } from 'office-ui-fabric-react/lib/Text';
+import { mergeStyles } from 'office-ui-fabric-react/lib/Styling';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+import { ApiReferencesTable, MEDIUM_GAP_SIZE, LARGE_GAP_SIZE } from './ApiReferencesTable';
+import {
+  IApiProperty,
+  IApiInterfaceProperty,
+  IApiEnumProperty,
+  IMethod,
+  IApiReferencesTableSetProps,
+  IEnumTableRowJson,
+  ITableRowJson
+} from './ApiReferencesTableSet.types';
+import { PropertyType } from '../../utilities/parser/index';
+
+export interface IApiReferencesTableSetState {
+  properties: Array<IApiProperty>;
+  showSeeMore: boolean;
+}
+
+const apiReferencesTableTopMargin = mergeStyles({
+  marginTop: '40px'
+});
+
+export class ApiReferencesTableSet extends React.Component<IApiReferencesTableSetProps, IApiReferencesTableSetState> {
+  constructor(props: IApiReferencesTableSetProps) {
+    super(props);
+
+    this._onClickSeeMore = this._onClickSeeMore.bind(this);
+
+    this.state = {
+      properties: this._generatePropertyArray(),
+      showSeeMore: false
+    };
+  }
+
+  public renderFirst(): JSX.Element | undefined {
+    if (this.state.properties.length >= 1) {
+      const item = this.state.properties[0];
+      return (
+        <ApiReferencesTable
+          key={item.propertyName}
+          name={item.propertyName}
+          title={item.title}
+          description={item.description}
+          extendsTokens={item.extendsTokens}
+          properties={item.property}
+          methods={item.methods}
+          renderAsEnum={item.propertyType === PropertyType.enum}
+          renderAsClass={item.propertyType === PropertyType.class}
+        />
+      );
+    }
+    return undefined;
+  }
+
+  public renderEach(): JSX.Element | undefined {
+    if (this.state.properties.length > 1) {
```

Comment:
Can this be combined with the previous method somehow (or move the common logic into a helper)? Code seems significantly duplicated.

Reply:
Combined the logic into `_renderReferencesTable()` :)